appendix_meta_rules.md

```diff
@@ -605,16 +620,90 @@ See Detection in [sigma specification](Sigma_specification.md)
 Example
 
 ```yaml
-title: Filter Administrator account 
-description: The valid administrator account start with adm_ 
-action: filter
-type: exclude
-rules: 
-    - 6f3e2987-db24-4c78-a860-b4f4095a7095 # Data Compressed - rar.exe
-    - df0841c0-9846-4e9f-ad8a-7df91571771b # Login on jump host
+title: Filter Administrator account
+description: The valid administrator account start with adm_
 logsource:
     category: process_creation
-    product: windows 
-selection:
-    User|startswith: 'adm_'
+    product: windows
```

Comment:
Just realized by doing a Github review myself: Do we need the logsources for filters? I think I just put them there by mistake/copy paste.

Reply:
I prefer it to be present to avoid filtering on fields that don't even exist in the original events.
It keeps a certain general sense.

Reply:
The downside would be, that you cannot use a global filter to whitelist an application for process_creation and file_event if filtering via Image was viable.

Reply:
Optional is fine to me.

Reply:
A filter would target fields that exist on all the referenced rules by definition. You can't filter a user if the field isn't there in the first place. So while you might say its implicit it's actually necessary because of field mapping. For example you might have username in process creation events as the username of the user executing the process but in a file event it might mean the owner of the file or something.